nano/node/bootstrap/bootstrap_connections.cpp

```diff
@@ -217,7 +217,7 @@ void nano::bootstrap_connections::populate_connections (bool repeat)
 			if (auto client = c.lock ())
 			{
 				new_clients.push_back (client);
-				endpoints.insert (client->socket->remote_endpoint ());
+				endpoints.insert (*client->socket->remote_endpoint ());
```

Comment:
Is this guaranteed to have a value? Isn't remote_endpoint the side that can switch to ANYADDR?